audio-worklet/src/bitcrusher-audio-worklet.js

```diff
@@ -40,15 +40,16 @@ registerProcessor('bitcrusher-audio-worklet',
   constructor(options) {
     super(options);
 
-    // Index_ and previousSample_ are member variables to handle block
-    // transitions.
+    // index_ and previousSample_ are member variables to handle block
+    // transitions. index_ refers to the current position in the input
+    // AudioBuffer, and previousSample_ refers to the corresponding sample.
```

Comment:
Nit: Renaming `index_` to `currentInputPosition_` would be nice because it explains what it really is.

But ok, if you leave it as is.